core/server/worker/src/main/java/alluxio/worker/block/UnderFileSystemBlockReader.java

```diff
@@ -233,9 +232,8 @@ public int transferTo(ByteBuf buf) throws IOException {
           mBlockWriter.transferFrom(bufCopy);
         }
       } catch (Exception e) {
-        LOG.warn("Failed to cache data read from UFS: %s", e.getMessage());
-        mBlockWriter.close();
-        mBlockWriter = null;
+        LOG.warn("Failed to cache data read from UFS in Alluxio: {}", e.getMessage());
```

Comment:
The two warnings are the same, so it's hard to know which one caused a given log message. Could we make them specific enough to be unique?

Reply:
I think we need to request space here too.

Reply:
done